thoughtful-post/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "thoughful-post-mailing",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "yarn mailing server build && cp -r ./emails/assets ./.mailing/public",
+    "dev": "yarn mailing",
+    "dev:local-assets": "npx mailing server scaffold && ln -sf ../../emails/assets ./.mailing/public && LOCAL_ASSETS=true yarn mailing preview"
```

Comment:
`npx mailing server scaffold`? I don't think that's a command.

Reply:
It isn't a command, it's a hack ensure that the .mailing directory exists. You can pass in any subcommand to the mailing server command, and unless it's `build` or `start` all it does is bootstrap the mailing directory.

Reply:
haha o cool